pom.xml

```diff
@@ -223,18 +223,4 @@
       </plugin>
     </plugins>
   </reporting>
-
-  <profiles>
-    <profile>
-      <!-- JDK 9+ has the "release" option to ensure the generated bytecode is
-       compatible with Java 8. -->
-      <id>compiler-release-8</id>
```

Comment:
This is now defined in the shared config pom.xml
https://github.com/googleapis/java-shared-config/commit/2468276145cdfe1ca911b52f765e026e77661a09#diff-9c5fb3d1b7e3b0f54bc5c4182965c4fe1f9023d449017cece3005d3f90e8e4d8L836